src/core/Player.ts

```diff
@@ -223,13 +223,19 @@ export default class Player {
   static extractNSigSourceCode(data: string): string | undefined {
     const match = data.match(/b=(?:a\.split\(|String\.prototype\.split\.call\(a,)\n?(?:""|\("",""\))\).*?\}return (?:b\.join\(|Array\.prototype\.join\.call\(b,)\n?(?:""|\("",""\))\)\}/s);
```

Comment:
Do we keep this here or should we always use the ast walker?

Reply:
I think we should always use the AST walker.